internal/provider/data_source_docker_registry_image_test.go

```diff
@@ -46,6 +48,7 @@ func TestAccDockerRegistryImage_private(t *testing.T) {
 func TestAccDockerRegistryImage_auth(t *testing.T) {
 	registry := "127.0.0.1:15000"
 	image := "127.0.0.1:15000/tftest-service:v1"
+	ctx := context.TODO()
```

Comment:
Maybe context.Background is better?

Reply:
So basically in all `_test.go` files use `context.BackGround()` Can do this

Reply:
I think so.
In my understanding, we should use `context.TODO()` when we plan to replace the context in future.
The purpose is to clarify as code that it should be replaced.

In case of test code, we don't have to replace the context so I think we shouldn't use `context.TODO()`.

Reply:
Fixed by 9aca4fe